app/components/teachers_index/table_section_component.rb

```diff
@@ -0,0 +1,32 @@
+module TeachersIndex
+  class TableSectionComponent < ViewComponent::Base
+    include GovukLinkHelper
+    include Pagy::Frontend
+    include Rails.application.routes.url_helpers
+
+    def initialize(teachers:, pagy:, status:, query:)
+      @teachers = teachers
+      @pagy = pagy
+      @status = status
+      @query = query
+    end
+
+  private
+
+    attr_reader :teachers, :pagy, :status, :query
+
+    def teachers_present?
+      teachers.any?
+    end
+
+    def empty_state_message
+      base_message = "No #{status} inductions found"
+
+      if query.present?
+        "#{base_message} matching \"<strong>#{query}</strong>\".".html_safe
```

Comment:
tag helper?  `tag.strong(query, class: 'govuk-!-font-weight-bold')`

Could put `empty_state_message` in a shared location unless the content is likely diverge